steps.sh

```diff
@@ -0,0 +1,10 @@
+git clone --depth 1 https://github.com/sunil-sahoo/opencart.git -b docker-deploy /project
+mv /project/upload/config-dist.php /project/upload/config.php 
```

Comment:
That must be part of Dockerfile, not this.
Of Makefile if you do "build"

Reply:
these steps are part of makefile, but as i haven't written the makefile so for reference i added steps.sh file. I'll remove this file after i add a makefile.